PROTOCOL.md

```diff
@@ -21,7 +21,7 @@ export interface OperationMessage {
 #### GQL_CONNECTION_INIT
 Client sends this message after plain websocket connection to start the communication with the server
 
-The server will response only with `GQL_CONNECTION_ACK` or `GQL_CONNECTION_ERROR` to this message.
+The server will response only with `GQL_CONNECTION_ACK` + `GQL_CONNECTION_KEEP_ALIVE` (if used) or `GQL_CONNECTION_ERROR` to this message.
```

Comment:
While very unlikely it would be possible when keep alive is in use for the server to send `GQL_CONNECTION_ACK` and then `GQL_CONNECTION_ERROR` if for some reason the `GQL_CONNECTION_KEEP_ALIVE` failed to send.

Reply:
if KEEP_ALIVE failed to send, ERROR will be failing as well...